packages/qwik/src/optimizer/src/qwik-binding-map.ts

```diff
@@ -30,5 +30,15 @@ export const QWIK_BINDING_MAP = {
         "platformArchABI": "qwik.win32-x64-msvc.node"
       }
     ]
+  },
+  "linux": {
+    "x64": [
+      {
+        "platform": "linux",
+        "arch": "x64",
+        "abi": "gnu",
+        "platformArchABI": "qwik.linux-x64-gnu.node"
+      }
+    ]
```

Comment:
I think these lines appeared after I ran `pnpm build.local` command. It's not related to the PR itself, so I'm not sure if I should keep it there.

Reply:
yes those are good :)